FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -341,6 +350,9 @@ export function SignUpFormContentComponent(props) {
           </div>
         </div>
       </form>
+      <Suspense fallback={<div>Loading...</div>}>
```

Comment:
RulesModal seems tiny, so I'm not sure whether we will win a lot with lazy. This could be applicable as an example

Reply:
According to statistics, nobody reads this text, so I used content placeholder  and this line   const RulesModal = React.lazy(() => import('./RulesModal'));